src/main/java/hudson/plugins/sonar/SonarPublisher.java

```diff
@@ -422,7 +414,10 @@ public boolean usesPrivateRepository() {
     return usePrivateRepository;
   }
 
-  @Extension(ordinal = 1000)
+  /**
+   * Optional because Maven plugin might not be available.
+   */
+  @Extension(optional = true, ordinal = 1000)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder annotation arguments to match the order of declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS3340)